main_test.go

```diff
@@ -23,17 +23,32 @@ func Test_RecepientsCheck(t *testing.T) {
 		denied   string
 		expected error
 	}{
+		{
+			name:   "with emails that are denied",
+			emails: []string{"delivery@example.com", "example@email.com", "<example@email.com>"},
```

Comment:
How will these be denied if there's no denyList defined?

Reply:
The wording was off, thanks!